liblangutil/ErrorReporter.h

```diff
@@ -147,6 +147,10 @@ class ErrorReporter
 	unsigned m_errorCount = 0;
 	unsigned m_warningCount = 0;
 
+	// currently only used within checkForExcessiveErrors(..)
```

Comment:
I don't think it is a clean strategy to treat that warning specially.

Reply:
Yeah I was also not very sure how to deal with this. At least for now this is the simplest solution that  I have found so far.